.hemtt/project.toml

```diff
@@ -0,0 +1,31 @@
+name = "Advanced Equipment"
+prefix = "ae3"
+mainprefix = "z"
+
+[properties]
+author = "y0014984|Wasserstoff"
+template = "cba"
+
+#folder_optionals = true
+#sig_version = 3
+
+[lint.sqf]
+enabled = true
+
+[version]
+git_hash = 0
```

Comment:
I thought that this could be ignored if you use `script_version.hpp` instead for versioning.

Reply:
ACE3 does this the same way.